tests/test_polyagamma.py

```diff
@@ -1,86 +1,87 @@
 import numpy as np
 import pytest
 
-from polyagamma import default_rng
-
-seed = 12345
-
-def test_default_rng():
-    rng = default_rng()
-    assert isinstance(rng, np.random.Generator)
-    pg = 'polyagamma'
-    assert (hasattr(rng, pg) and callable(getattr(rng, pg)))
+from polyagamma import polyagamma
 
 
 def test_polyagamma():
-    rng = default_rng(seed)
-    assert len(rng.polyagamma(size=5)) == 5
-    assert rng.polyagamma(size=(5, 2)).shape == (5, 2)
+    rng = np.random.default_rng(1)
```

Comment:
Since it looks like you're now only using this for drawing a couple of the input parameters, I'd probably just explicitly specify those input parameters. If you want to make sure that all of the `polyagamma()` calls are testing the exact same thing every time, I'd consider wrapping this with `functools.partial()`: `rng_polyagamma = functools.partial(polyagamma, random_state=rng)` and using `rng_polyagamma()` below (except for the explicit reproducibility tests down at the bottom).

Otherwise, LGTM!

Reply:
The partial function idea is great. I just pushed the changes. Time for a merge. Thanks for the review!